rgw/v2/tests/s3_swift/configs/test_object_lock.yaml

```diff
@@ -0,0 +1,12 @@
+# upload type: non multipart
+# script: test_object_lock.py
```

Comment:
It's always helpful if could add meaning comment messages.

Reply:
Done. Updated